tools/rpm2img

```diff
@@ -60,36 +60,36 @@ VERITY_HASH_BLOCK_SIZE=4096
 # for the boot partition, where we set gptprio bits in the GUID-specific use
 # field, but we might as well do it for all of them.
 BOTTLEROCKET_BOOT_TYPECODE="6b636168-7420-6568-2070-6c616e657421"
-EFI_SYSTEM_TYPECODE="C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
 BOTTLEROCKET_ROOT_TYPECODE="5526016a-1a97-4ea4-b39a-b7c8c6ca4502"
 BOTTLEROCKET_HASH_TYPECODE="598f10af-c955-4456-6a99-7720068a6cea"
 BOTTLEROCKET_RESERVED_TYPECODE="0c5d99a5-d331-4147-baef-08e2b855bdc9"
 BOTTLEROCKET_PRIVATE_TYPECODE="440408bb-eb0b-4328-a6e5-a29038fad706"
 BOTTLEROCKET_DATA_TYPECODE="626f7474-6c65-6474-6861-726d61726b73"
+BIOS_TYPECODE="ef02"
 
-if [[ "${ARCH}" == "x86_64" ]]; then
-  FIRM_NAME="BIOS-BOOT"
-  BOTTLEROCKET_FIRM_TYPECODE="ef02"
-else
-  FIRM_NAME="EFI_BOOT"
-  BOTTLEROCKET_FIRM_TYPECODE="${EFI_SYSTEM_TYPECODE}"
-fi
+# We'll eventually want to support an active/passive scheme for EFI partitions,
```

Comment:
It took me a bit of offline discussion to understand what's going on with the EFI partitions, and it boiled down to one key point that might be worth putting into a comment in addition to this explanation - we'll be using the grub stage 2 binary from the EFI partition in the _first_ partition set regardless of the _target_ partition set.  It'll then use the counters to continue booting into the target partition set.  I was confused because of the single BIOS boot partition versus the two EFI partitions, because in all(?) other cases we only use artifacts from the target set.

Reply:
I've expanded the commentary and changed the variable names to avoid confusion with the A/B partition sets.